client/src/components/CategoryList.js

```diff
@@ -0,0 +1,110 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+import { ReactComponent as Bamboo } from "../assets/icons/bamboo.svg";
+import { ReactComponent as Travel } from "../assets/icons/travel.svg";
+import { ReactComponent as Bath } from "../assets/icons/bathroom.svg";
+import { ReactComponent as Gift } from "../assets/icons/gift.svg";
+import { ReactComponent as Household } from "../assets/icons/household.svg";
+import { ReactComponent as Ocean } from "../assets/icons/ocean.svg";
+import { ReactComponent as Reuse } from "../assets/icons/reuse.svg";
+import { ReactComponent as Tree } from "../assets/icons/tree.svg";
+
+const ListContainer = styled.ul`
+  list-style: none;
+  display: grid;
+  gap: 40px;
+  grid-template-columns: 1fr 1fr 1fr 1fr;
+  grid-template-rows: 1fr 1fr;
+`;
+
+const BambooIcon = styled(Bamboo)`
+  height: 60px;
+  width: 60px;
+  fill: ${(props) =>
+    props.active ? "var(--icon-active-color)" : "var(--icon-inactive-color)"};
+`;
+const TravelIcon = styled(Travel)`
+  height: 60px;
+  width: 60px;
+  fill: ${(props) =>
+    props.active ? "var(--icon-active-color)" : "var(--icon-inactive-color)"};
+`;
+const BathIcon = styled(Bath)`
+  height: 60px;
+  width: 60px;
+  fill: ${(props) =>
+    props.active ? "var(--icon-active-color)" : "var(--icon-inactive-color)"};
+`;
```

Comment:
concerning your question in your PR-Description, there is a way to do this via styled.css

i couldnt find it in the docs, maybe somebody else can give you a hint :)
otherwise just try it out
`
const Icon = styled.css`
  height: 60px;
  width: 60px;
  fill: ${(props) =>
    props.active ? "var(--icon-active-color)" : "var(--icon-inactive-color)"};
`;
const TravelIcon=styled(Icon) ...

``

Reply:
Thanks! This definitely seems like a useful suggestion, however I can't get it to work with the current code...
I will try again tomorrow though ;-)